src/Illuminate/Database/Console/PruneCommand.php

```diff
@@ -49,10 +51,14 @@ public function handle(Dispatcher $events)
         }
 
         if ($this->option('pretend')) {
+            $events->dispatch(new ModelsPruningStarted($models->toArray()));
```

Comment:
@ziadoz Would it be an idea to pass the pretending flag from the command into the starting and ending events respectively?

Reply:
@morloderex Yes, I think it would be a good idea, but it would be inconsistent with the existing migration events. It might be worth implementing in a separate PR though.